packages/babel-runtime/scripts/build-dist.js

```diff
@@ -88,9 +115,10 @@ each(helpers.list, function (helperName) {
 
   // compat
   var helperAlias = _.kebabCase(helperName);
-  writeFile("helpers/_" + helperAlias + ".js", buildHelper(helperName));
-  writeFile("helpers/" + helperAlias + ".js", buildHelper(helperName));
+  var content = "module.exports = require(\"./" + helperName + ".js\");";
+  writeFile("helpers/_" + helperAlias + ".js", content);
+  if (helperAlias !== helperName) writeFile("helpers/" + helperAlias + ".js", content);
```

Comment:
As there will be now a lot less duplication, perhaps you could consider removing built helpers from [.gitignore](https://github.com/babel/babel/blob/e02606b/.gitignore#L15-L16) and checking them in repo, so that lerna could find out when babel-runtime package has changed? (It uses git diff, so doesn't currently know when helpers have changed: https://github.com/sebmck/lerna/blob/29455d7/lib/commands/publish.js#L109-L111). Would make referring to babel-runtime source easier too.